src/components/LeaderboardPage/RankingList.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box, Avatar, Typography, Paper } from '@mui/material';
+import React from 'react';
+
+import ranking_1 from '../../assets/ranking_one.svg';
+import ranking_3 from '../../assets/ranking_three.svg';
+import ranking_2 from '../../assets/ranking_two.svg';
+
+import { HighlightRow, SyneTypography } from './LeaderboardPage.styles';
+
+import type { LeaderboardUser } from './LeaderboardPage.types';
+
+export const RankingList: React.FC<{ users: LeaderboardUser[]; currentUserId?: string }> = ({
+  users,
+  currentUserId,
+}) => {
+  return (
+    <Box sx={{ display: 'grid', gap: 1 }}>
+      {users.map((u) => {
+        let rankImg: string | null = null;
+        if (u.rank === 1) rankImg = ranking_1;
+        else if (u.rank === 2) rankImg = ranking_2;
+        else if (u.rank === 3) rankImg = ranking_3;
+
+        const content = (
+          <Paper variant="outlined" sx={{ p: 1 }} data-testid={`card-${u.username}`}>
+            <Box
+              sx={{
+                display: 'flex',
+                alignItems: 'center',
+                justifyContent: 'space-between',
+                gap: 2,
+              }}
+            >
+              <Box sx={{ display: 'flex', alignItems: 'center', gap: 2 }}>
+                <Avatar src={u.avatarUrl} />
```

Comment:
Avatar is rendered without an alt attribute. Avatars that are images should include alt text for screen readers or be marked as decorative when appropriate.

**Sugerencia:**
```typescript
Add an alt attribute or aria-label to the Avatar, e.g. <Avatar src={u.avatarUrl} alt={u.fullName} />. If the avatar is purely decorative, add alt="" and ensure the surrounding markup provides the user's name in text.
```